app/views/csvs/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Request a CSV Report</h1>
+
+<section class="clearfix">
+  <div class="csv-deploy-filters clearfix">
+    <%= form_tag csvs_path do %>
+      <div><strong>Deploy reports</strong></div>
+      <div>
+        <%= label_tag :start_date, "Start Date:" %>
+        <%= date_field_tag :start_date, Date.today.prev_year.strftime("%m/%d/%Y"), id: "datepicker1" %>
```

Comment:
... might be better to go with %Y-%m-%d ...

On Wed, Mar 23, 2016 at 5:03 PM, Roman Sandler <notifications@github.com>
wrote:

> In app/views/csvs/new.html.erb
> <https://github.com/zendesk/samson/pull/766#discussion_r57257656>:
>
> > @@ -0,0 +1,37 @@
> > +<h1>Request a CSV Report</h1>
> > +
> > +<section class="clearfix">
> > +  <div class="csv-deploy-filters clearfix">
> > +    <%= form_tag csvs_path do %>
> > +      <div><strong>Deploy reports</strong></div>
> > +      <div>
> > +        <%= label_tag :start_date, "Start Date:" %>
> > +        <%= date_field_tag :start_date, Date.today.prev_year.strftime("%m/%d/%Y"), id: "datepicker1" %>
>
> fyi this date format is not standard across all locales :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/766/files/fbb4d120d26ded64e1ff88a3485de7c2957cd7fd#r57257656>
>